src/constants.js

```diff
@@ -21,13 +21,13 @@ const ERRORS = {
     "PR title does not match PR title convention: type: subject or type(scope): subject",
     `For \`type\`, use \`type:\` or \`type(scope):\`, where \`type\` is one of ${displayTypes}`,
     `If present, \`scope\` must be one of ${displayScopes} or \`<displayName> Node\``,
-    "For the subject, mind the whitespace, start with lowercase and omit final period",
+    "For the subject, mind the whitespace, start with uppercase and omit final period",
   ].join(". "),
   TICKET_NUMBER_PRESENT: "PR title must not contain a ticket number",
   TYPE_NOT_FOUND: `Failed to find \`type\` in PR title. Expected one of ${displayTypes}`,
   INVALID_TYPE: `Unknown \`type\` in PR title. Expected one of ${displayTypes}`,
   INVALID_SCOPE: `Unknown \`scope\` in PR title. Expected one of ${displayScopes} or \`<displayName> Node\``,
-  LOWERCASE_INITIAL_IN_SUBJECT: "First char of subject must be uppercase",
+  UPPERCASE_INITIAL_IN_SUBJECT: "First char of subject must be uppercase",
```

Comment:
From what I see all constants here are error names, so no need to rename it?

Reply:
You're right. I'll fix it.